lib/grape/endpoint.rb

```diff
@@ -248,6 +248,8 @@ def run
 
         run_filters befores, :before
 
+        raise Grape::Exceptions::MethodNotAllowed, header.merge('Allow' => env[Grape::Env::GRAPE_METHOD_NOT_ALLOWED]) if env[Grape::Env::GRAPE_METHOD_NOT_ALLOWED]
```

Comment:
rubocop was fine with both(the parens around it quieted it).  But it caused the class to have too many lines.  option 2 seems to take care of that restraint, but either that limit will need to be raised soon or....

and the 3rd instance of `env[Grape::Env::GRAPE_METHOD_NOT_ALLOWED]` is actually completely redundant since we are raising an exception.  So I removed it with my latest change.